### PR TITLE
tools/etcd-dump-logs: Fixed default values for -entry-type flag

### PR DESCRIPTION
**Pull request by vimalk78**
The tool takes default values but it was not visible which default
values were taken. Added default values in proper place, and added a
newline at the end of output.

Fixed failing test case because of this change.

Please read https://github.com/etcd-io/etcd/blob/master/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

**Comment by vimalk78** [comment]
@gyuho , @xiang90 PTAL

File: tools/etcd-dump-logs/main.go [hunk 1]
```diff
@@ -38,10 +38,14 @@ import (
...
 
+const (
+	defaultEntryTypes string = "Normal,ConfigChange"
+)
+
 func main() {
 	snapfile := flag.String("start-snap", "", "The base name of snapshot file to start dumping")
 	index := flag.Uint64("start-index", 0, "The index to start dumping")
-	entrytype := flag.String("entry-type", "", `If set, filters output by entry type. Must be one or more than one of:
+	entrytype := flag.String("entry-type", defaultEntryTypes, `If set, filters output by entry type. Must be one or more than one of:
```
Comment (xiang90) [inline tools/etcd-dump-logs/main.go]: comment that the default type is Normal and ConfigChange.

**Comment by xiang90** [comment]
lgtm after fixing the nit.

File: tools/etcd-dump-logs/main.go (hunk 1 above)
Comment (vimalk78) [reply tools/etcd-dump-logs/main.go]: updated. PTAL

**Comment by vimalk78** [comment]
The test cases in `tools/etcd-dump-logs` fail locally if timezone is other than Seattle :)
Because the `etcd-dump-logs/expectedoutput/*` has saved timestamp in PST.

Perhaps this should be fixed, and additionally the tool should be able to dump data with a different timezone.

**Comment by jingyih** [comment]
cc @wenjiaswe

**Pull request merged**